src/reader.rs

```diff
@@ -935,25 +939,32 @@ pub fn reader_change_cursor_end_mode(end_mode: CursorEndMode) {
     }
 }
 
-fn check_autosuggestion_enabled(vars: &dyn Environment) -> bool {
-    vars.get(L!("fish_autosuggestion_enabled"))
+fn check_var(vars: &dyn Environment, name: &wstr, default: bool) -> bool {
```

Comment:
I'd probably call this `check_bool_var` but I guess it doesn't matter, it's obvious enough.
(it always trips me up that I can't use `set fish_autosuggestion_enabled false` but that's a different issue).